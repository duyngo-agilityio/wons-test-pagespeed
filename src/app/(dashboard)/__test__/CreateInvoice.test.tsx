import { render, waitFor } from '@testing-library/react';

// Pages
import CreateInvoicePage from '@/app/(dashboard)/create-invoice/page';

jest.mock('@/ui', () => ({
  CreateInvoice: () => <div data-testid="create-invoice">Create Invoice</div>,
}));

describe('CreateInvoice Page render', () => {
  it('should render and match with snapshot', async () => {
    const { container } = render(<CreateInvoicePage />);

    await waitFor(() => {
      expect(container).toBeInTheDocument();
    });

    expect(container).toMatchSnapshot();
  });
});
