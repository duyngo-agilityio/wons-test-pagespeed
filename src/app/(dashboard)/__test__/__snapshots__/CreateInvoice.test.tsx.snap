// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateInvoice Page render should render and match with snapshot 1`] = `
<div>
  <section>
    <div
      class="base:mb-6 md:mb-7.5"
    >
      <h2
        aria-level="2"
        class="font-bold font-dm-sans text-blue-800 dark:text-white text-5xl w-full"
      >
        Create New Invoice
      </h2>
    </div>
    <div
      data-testid="create-invoice"
    >
      Create Invoice
    </div>
  </section>
</div>
`;
